tutordiscovery/patches/local-docker-compose-services

```diff
@@ -5,6 +5,7 @@ discovery:
   restart: unless-stopped
   volumes:
     - ../plugins/discovery/apps/settings/tutor:/openedx/discovery/course_discovery/settings/tutor:ro
+    - ../../data/discovery/media:/openedx/discovery/course_discovery/media
```

Comment:
do we need this in k8s?

Reply:
I was thinking the same thing, but we don't support PVC for media files in Kubernetes. Users are expected to make use of MinIO/S3. That being said, we should make sure that running tutor-discovery+tutor-minio does result in storing media files in minio...

Reply:
I have tested it by installing/enabling tutor-minio plugin with tutor-discovery plugin, minio storage doesn't support/store discovery media. Created an [issue](https://github.com/overhangio/tutor-minio/issues/41) in minio plugin. I will work on that but meanwhile we can merge this PR.